api/lib/domain/services/user-reconciliation-service.js

```diff
@@ -63,10 +50,10 @@ async function checkIfStudentIsAlreadyReconciledOnTheSameOrganization(matchingSc
   if (!_.isNil(matchingSchoolingRegistration.userId))  {
     const userId = matchingSchoolingRegistration.userId ;
     const user = await userRepository.getUserAuthenticationMethods(userId);
-    const authentificationMethod = getUserAuthenticationMethodWithObfuscation(user);
+    const authentificationMethod = await getUserAuthenticationMethodWithObfuscation(user);
```

Comment:
Du coup  pas besoin du `await` ici 
```suggestion
    const authentificationMethod = getUserAuthenticationMethodWithObfuscation(user);
```